addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -232,7 +232,7 @@
                     <field name="manager_id"/>
                 </xpath>
                 <xpath expr="//group[@name='groupby']" position="inside">
-                    <filter string="Project" name="groupby_project" domain="[]" context="{'group_by': 'project_id'}"/>
+                    <filter string="Source Project" name="groupby_project" domain="[]" context="{'group_by': 'project_id'}"/>
```

Comment:
I think we should not alter that line since the search view should only be used in timesheets app.

Reply:
> I think we should not alter that line since the search view should only be used in timesheets app.

To prevent duplication when the user selects a project from the custom group, I have attached the screenshot in the task chatter.

Reply:
Why would the user like to select the other project in timesheets? 🤔 

Let's ask the opinion of @aju-odoo but to me, it is not needed to alter the views in timesheets, we alter the project label to avoid having 2 Project fields in analytic views but not really in timesheets.